app/decorators/offence_summary_decorator.rb

```diff
@@ -2,12 +2,22 @@
 
 class OffenceSummaryDecorator < BaseDecorator
   def plea_sentence
-    return t('generic.not_available') if plea&.value.blank?
-    return_plea plea
+    return t('generic.not_available') if pleas.blank?
+    return pleas unless pleas.is_a?(Enumerable)
+
+    safe_join(plea_sentences, tag.br)
   end
 
```

Comment:
Looks good to me. I noticed that there's no unit test for the plea sentence / OffenceSummaryDecorator

Reply:
Its integrated tested within the multiple different tests we have so i dont think we need to go to the nth degree testing this.